bin/zopen-build

```diff
@@ -1847,6 +1847,107 @@ zz
   cp "${ZOPEN_ROOT}/install/metadata.json" "${ZOPEN_INSTALL_DIR}/metadata.json"
 }
 
+set_install_caveats() 
+{
+  package=${ZOPEN_NAME}
+  name=$(echo $package | cut -d "-" -f 1)
+  prefix="$1"
+
+  # Add "GNU " prefix to tool name only if prefix is "g"
+  case "$prefix" in
+      g) tool_label="GNU $name" ;;
+      *) tool_label="$name" ;;
+  esac
+
+  cat <<ZZ
+
+$tool_label have been installed with the prefix "$prefix" under bin/ to avoid collision with z/OS /bin/ tools.
```

Comment:
```suggestion
$tool_label has been installed with the prefix "$prefix" under bin/ to avoid collision with z/OS /bin/ tools.
```
So it works for both singular and plural